controllers/install/common_helpers.go

```diff
@@ -54,3 +60,34 @@ func AllLabels(name string, labels map[string]string) map[string]string {
 	baseLabels = MergeMaps(baseLabels, identityLabels)
 	return baseLabels
 }
+
+// waitForJob will wait for some resolution of the job. Provide context with timeout if needed.
+func waitForJob(ctx context.Context, cl client.Client, job *batchv1.Job) (err error) {
```

Comment:
Do we want to export the function?  

And do we want some unit tests around this?  We have mocks for job so it wouldn't be so bad to mock this.

Reply:
Good idea - I think I may want to use this func when I do my Armada migration stuff - exporting it would be great.

Reply:
I can add a unit test

Reply:
i wouldn't export it unless we need it outside of the install module